### DISCUSSION TITLE
Add conversion value to events

### DISCUSSION THREAD

**Discussion by Thibautds** [plausible/analytics#2547] [Anything Else]
It would be nice if we can add a conversion value to our custom events.
We can use this for basic ecommerce tracking and also to calculate the total conversion value of our events.

**Comment by metmarkosaric** [comment]
hi @Thibautds! you could use this with [custom properties for custom events](https://plausible.io/docs/custom-event-goals#using-custom-props). here's an example of how it looks like in your dashboard when you send conversion values as custom properties: https://plausible.io/elementary.io?period=month&goal=Payment. this might work for you for now while we're working on building more advanced monetary values feature in the future #256

**Comment by metmarkosaric** [comment]
**Update:** We've now opened up a preview version of this to everyone. You can now assign monetary values to custom events and track the revenue attribution of your ecommerce store. Instructions at https://plausible.io/docs/ecommerce-revenue-tracking
